app/views/kaminari/blacklight/_paginator.html.erb

```diff
@@ -0,0 +1,27 @@
+<%# Copied from Blacklight core and modified to change positioning of %>
+<%# Next button per RDR-231. %>
+<%# Last checked for upstream updates: Hyrax 2.5.1 / Blacklight 6.14 %>
```

Comment:
The 'develop' branch is now using Hyrax 2.6.0.  Not a big deal but you might want to check this against Hyrax 2.6.0 rather than 2.5.1 and update the comment accordingly.

Reply:
@coblej Good catch!